packages/quantic/force-app/main/default/lwc/quanticSearchBoxSuggestionsList/quanticSearchBoxSuggestionsList.js

```diff
@@ -41,87 +89,205 @@ export default class QuanticSearchBoxSuggestionsList extends LightningElement {
   @api
   selectionDown() {
     this.selectionIndex++;
-    if (this.selectionIndex >= this.suggestions.length) {
+    if (this.selectionIndex >= this.allOptions.length) {
       this.selectionIndex = 0;
     }
+    return {
+      id: this.allOptionsHTMLElements[this.selectionIndex].getAttribute('id'),
+      value: this.allOptions[this.selectionIndex].rawValue,
+    };
   }
 
   /**
    * Return the currently selected suggestion.
-   * @returns {Suggestion}
+   * @returns {Object}
    */
   @api
   getCurrentSelectedValue() {
-    if (this.selectionIndex > -1) {
-      return this.suggestions[this.selectionIndex];
+    if (this.allOptions?.[this.selectionIndex]) {
+      const {rawValue, isClearRecentQueryButton, isRecentQuery} =
+        this.allOptions[this.selectionIndex];
+      return {
+        value: rawValue,
+        isClearRecentQueryButton,
+        isRecentQuery,
+      };
     }
-    return undefined;
-  }
-
-  /**
-   * Reset the current selection.
-   */
-  @api
-  resetSelection() {
```

Comment:
If we can keep this it would be great. Even if we don't use it ourselves.

Mark as @deprecated, and make a jira to remove it for v3

Reply:
added back and marked as deprecated.